js/lib/bandstructure.js

```diff
@@ -176,7 +176,7 @@ BandPlot.prototype.addBandStructure = function (bandsData, colorInfo) {
 BandPlot.prototype.addDos = function (dosData) {
     this.dosData = dosData;
 
-    var Index = 1 + dosData['pdos'].length;
+    var Index = 1 + dosData['dos'].length;
```

Comment:
I guess for java the index starts from 1? Can you add a comment in the code?

Reply:
> I guess for java the index starts from 1? Can you add a comment in the code?

Thanks for spotting this.  This is for previous total dos + pdos. Now, we should delete the 1, since all data is in the dos dict.

Reply:
Can you attach the screenshot? Thanks.

Reply:
> Can you attach the screenshot? Thanks.

what screenshot?

Reply:
Pdos figure after the change of this PR. We don't have test for it yet.

Reply:
> Pdos figure after the change of this PR. We don't have test for it yet.

![image](https://user-images.githubusercontent.com/56022756/186862048-830902c4-c98a-4459-a3cc-c6d7b583a6d5.png)

Reply:
The plot is not very clear I think. Is it support transparent background (for example by not setting the value for `backgroundColor`)? We need one more option to set it.

Reply:
I'd appreciate if you  can drawing a plot where only the Total DOS is filled but other states are just solid lines.

Reply:
As we discuss offline, we will merge this PR first and improve the widget so that the transparency can be set for the plot of each orbital.